tests/client_tests/go/basic_queries.go

```diff
@@ -0,0 +1,70 @@
+
+package main
+
+import (
+    "log"
+    "flag"
+    "fmt"
+    "github.com/jackc/pgx"
+)
+
+
+func main() {
+    hosts := flag.String("hosts", "", "CrateDB hostname")
+    port := flag.Int("port", 5432, "CrateDB postgres port")
+    flag.Parse()
+
+	pgxConfig := pgx.ConnConfig{
```

Comment:
indention off?

Reply:
ah yep I run that part through gofmt and didn't notice tabs vs. spaces difference in my editor =)